core/src/main/java/fr/sncf/osrd/api/PathfindingRoutesEndpoint.java

```diff
@@ -81,6 +81,8 @@ public Response act(Request req) {
             return new RsJson(new RsWithBody(adapterResult.toJson(res)));
         } catch (Throwable ex) {
             return ExceptionHandler.handle(ex);
+        } finally {
```

Comment:
writting to stderr can be very slow. I think doing so only really makes sense for offline stuff, not API mode